tests/components/freebox/const.py

```diff
@@ -93,75 +93,177 @@
     {
```

Comment:
May be you can add disks in stead of changing current test data? This ensures the current code is tested correctly.

Reply:
Sorry, i re-read your comment; i would like to avoid crafting test data (i.e. guess how the router should look like if it had both the initial configuration and the new one). Maybe i should just create a dedicated const-raid.py module ?

Reply:
Also, for the record there are no tests on the current data.